tool/find.go

```diff
@@ -286,23 +285,11 @@ func (f *findT) readManifests(stdout io.Writer) {
 					if _, ok := f.tableMeta[nf.Meta.TableNum]; !ok {
 						f.tableMeta[nf.Meta.TableNum] = nf.Meta
 					}
-				}
-				for _, bf := range ve.NewBlobFiles {
-					if _, ok := blobMetas[bf.FileNum]; !ok {
-						blobMetas[bf.FileNum] = struct{}{}
-					}
+					f.blobRefsMap[nf.Meta.TableNum] = &nf.Meta.BlobReferences
```

Comment:
the issue that exists in `findAndReadManifests` also existed in `readManifests` -- i think we can do something like this for a quick fix (and we can do the subcommand suggestion later), but this works under the assumption that manifests will be read in order from least recent to most recent version edits (i think) https://github.com/cockroachdb/pebble/blob/d742eb87709bc5e19f29a467f4df174ae6fbe5d1/tool/find.go#L220

cc: @jbowens @RaduBerinde what do u all think?

Reply:
This looks good to me